src/main/java/com/hydratereminder/HydrateReminderPlugin.java

```diff
@@ -318,13 +318,13 @@ public String getTimeDisplay(Duration duration)
 		final StringBuilder timeDisplayBuilder = new StringBuilder();
 		if (hours > 0)
 		{
-			timeDisplayBuilder.append(hours != 1 ? hours + " hours " : hours + " hour ");
+			timeDisplayBuilder.append(hours + " hour").append(hours > 1 ? "s " : " ");
 		}
 		if (minutes > 0 || hours > 0)
 		{
-			timeDisplayBuilder.append(minutes != 1 ? minutes + " minutes " : minutes + " minute ");
+			timeDisplayBuilder.append(minutes + " minute").append(minutes > 1 ? "s " : " ");
 		}
-		timeDisplayBuilder.append(seconds != 1 ? seconds + " seconds" : seconds + " second");
+		timeDisplayBuilder.append(seconds + " second").append(seconds > 1 ? "s " : "");
```

Comment:
For minutes and seconds, we also want to add the `s` if they are equal to 0.

For example, `1 hour 0 minutes 0 seconds`.

Reply:
Done @jmakhack